presto-main/src/main/java/com/facebook/presto/operator/aggregation/GenericAccumulatorFactory.java

```diff
@@ -791,6 +823,11 @@ public void prepareFinal()
             delegate.prepareFinal();
         }
 
+        private static RunLengthEncodedBlock createNullBlock(Type type, int positionCount)
+        {
+            return new RunLengthEncodedBlock(type.createBlockBuilder(null, 1).appendNull().build(), positionCount);
+        }
```

Comment:
We can inline this function as it is only used by one place. Also, let's use `RunLengthEncodedBlock.create` which is simpler then `new RunLengthEncodedBlock`.